crates/bevy_math/src/curve/mod.rs

```diff
@@ -1015,6 +1305,51 @@ mod tests {
         assert_eq!(mapped_curve.domain(), interval(0.0, 1.0).unwrap());
     }
 
+    #[test]
+    fn reverse() {
```

Comment:
This test and the following ones are not really testing the `domain()` method.

By doing something like this, you'd be checking both the `sample_unchecked` and `domain()` methods :
```rust
assert_eq!(rev_curve.sample(0.0), Some(1.0 * 3.0 + 1.0));
assert_eq!(rev_curve.sample(0.5), Some(0.5 * 3.0 + 1.0));
assert_eq!(rev_curve.sample(1.0), Some(0.0 * 3.0 + 1.0));
assert_eq!(rev_curve.sample(1.1), None);
```